### PR TITLE
feat(deploy/ottelcol/logs): expose json_logs for json format

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Signed-off-by: Dominik Rosiek <drosiek@sumologic.com>

###### Description

Fill in your description here.

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by astencel-sumo — APPROVED** [review]
Why do we need this? For visibility?

The customer could add these options in their own `values.yaml` anyway, right?

**Comment by sumo-drosiek** [comment]
This is exposed by fluentd. It will simplify documentation and transition

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -3016,6 +3016,9 @@ otelcol:
         exporters:
           sumologic/containers:
             log_format: json
+            json_logs:
+              add_timestamp: true
+              tiestamp_key: timestamp
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:3021]: typo 🐛 

```suggestion
              timestamp_key: timestamp
```

File: deploy/helm/sumologic/values.yaml [hunk 2]
```diff
@@ -3048,6 +3051,9 @@ otelcol:
               - statefulset
           sumologic/systemd:
             log_format: json
+            json_logs:
+              add_timestamp: true
+              tiestamp_key: timestamp
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:3056]: typo 🐛 

```suggestion
              timestamp_key: timestamp
```

**Review by pmalek-sumo — CHANGES_REQUESTED** [review]

**Review by pmalek-sumo — APPROVED** [review]
What about the log key?

**Comment by sumo-drosiek** [comment]
@pmalek-sumo I didn't saw it exposed in fluentd, so no need to expose it here

**Pull request merged**